hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/committers.md

```diff
@@ -650,7 +639,14 @@ Conflict management is left to the execution engine itself.
 </property>
```

Comment:
full stop/question mark on line 632. little confusing to read currently. also i don't see a config option for ` fs.s3a.committer.staging.uuid` in staging committer options list

Reply:
Good call out, I had a look and I think that option should actually be just `fs.s3a.committer.uuid` - no `staging`. I'll update it.

The code in `AbstractS3ACommitter` checks for `fs.s3a.committer.uuid` and the Spark UUID before failing fast.

https://github.com/apache/hadoop/blob/e199da3fae1c82e87f88c8c50f6a96c6515e2edd/hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/commit/AbstractS3ACommitter.java#L1341-L1360